package.json

```
@@ -1,11 +1,11 @@
 {
     "name": "ansys_sphinx_theme",
     "repository": "https://github.com/ansys/ansys-sphinx-theme",
-    "description": "",
-    "main": "",
+
     "scripts": {
-        "build": "echo 'No build step needed' "
-    },
-    "dependencies": {},
-    "devDependencies": {}
+    "build":  "sass src/ansys_sphinx_theme/assets/styles/ansys-sphinx-theme.scss  src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/styles/ansys-sphinx-theme.css"
+},
+   "dependencies": {
+    "sass": "^1.43.4"
```

Can we go with the latest one?

```suggestion
    "sass": "^1.85.1"
```

---

sure we can, adding that and i was trying to deprecate the warning in my local